compute_sdk/globus_compute_sdk/sdk/utils/gare.py

```diff
@@ -0,0 +1,22 @@
+import typing as t
+
+from globus_sdk import GlobusAPIError, GlobusApp
+from globus_sdk.gare import to_gares
+
+
+def gare_handler(app: GlobusApp, f: t.Callable, *args, **kwargs):
+    try:
+        return f(*args, **kwargs)
+    except GlobusAPIError as e:
+        gares = to_gares([e, e.raw_json or {}])
+        if not gares:
```

Comment:
Och, missed this.  We need to pick one or the other, or we'll have an extra (and redundant) GARE login.  When the Globus SDK learns to properly parse the exception, we can switch to that.  But at this point, we should rely on the `.raw_json` property:
```python
gares = to_gares([e.raw_json or {}])
```
That took me a while to track down earlier.  Woof.

Reply:
fixed!